mapiproxy/servers/default/emsmdb/oxcfxics.c

```diff
@@ -2524,6 +2534,7 @@ _PUBLIC_ enum MAPISTATUS EcDoRpc_RopSyncImportDeletes(TALLOC_CTX *mem_ctx,
 	struct GUID				replica_guid;
 	uint16_t				repl_id;
 	struct mapi_SBinaryArray		*object_array;
+	uint64_t				*object_ids;
```

Comment:
Initialize object_ids to NULL. If you apply the goto end: when checking openchangedb_get_mailboxReplica, it will lead to inconsistent behavior.